TACC/Core-Portal
# Bug/WP-779--[Regression] File preview is smaller compared to production

## Overview

File preview is smaller compared to production

## Related

* [WP-779](https://tacc-main.atlassian.net/browse/WP-779)

## Changes
- Bootstrap embeded responsive 4x3 to ratio 4x3

## Testing

1. Upload an image to Datafiles
2. Ensure File Preview matches production sizing

## UI
Before
![image](https://github.com/user-attachments/assets/6312e151-3b9f-45fd-a12f-c0af1ff60b43)

After
![image](https://github.com/user-attachments/assets/f8843df2-a2f6-45a1-9328-7cbdccde4600)

Production
![image](https://github.com/user-attachments/assets/acb8abd6-f46d-4904-ba71-df8c77d836ca)

## Notes
- This was due to the Bootstrap update from 4.3 to 5.1[^1]
- Noticed a styling difference in the close X

[^1]: "[Responsive embed helpers [renamed]](https://getbootstrap.com/docs/5.0/migration/#helpers)"
